### PR TITLE
Adding installation for Cloudflare Warp

### PR DESCRIPTION
**Pull request by julysfire**
Adding install instructions for Cloudflare Warp.  

Instructions include information about what Cloudflare Warp is and how it can help, install instructions for Windows including how to turn the service on after installation, Linux install instructions and pictures for instructions.

Also please ignore that I forgot to add a commit message, this is the first time I've used Gitbook.

### CODE REVIEW

File: connectivity/cloudflare-warp.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    visible: true
+  outline:
+    visible: true
+  pagination:
+    visible: true
+---
+
+# Cloudflare Warp
+
+Cloudflare Warp is a recursive DNS that tunnels your connection to the nearest Cloudflare data center where it can then be routed to the desired destination.  Instead of bouncing through a VPN or an ISP DNS service, Cloudflare will do the routing.  This can increase your connectivity to the game's servers.\
```
Comment (Phyrok) [inline connectivity/cloudflare-warp.md:22]: Remove \

File: connectivity/cloudflare-warp.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+1. In your preferred browser, navigate to [https://1.1.1.1](broken-reference)
+2. Choose your operating system to start the download.
+
+<figure><img src="../.gitbook/assets/cloudflare1.PNG" alt="" width="563"><figcaption></figcaption></figure>
+
+**Windows**
+
+1. Once complete, navigate to your downloads folder (C:\Users\\\<YourUsername>\Downloads).  You should see a file called "Cloudflare\_WARP\_Release-x64.msi".  Double click this to install.
+2. In the installer, click Next and then "Install".  When complete, click "Finish".
+3. On your System Tray in the bottom right, you should see a gray cloud icon.  If you don't, you might have to click the up arrow to show more icons.\
```
Comment (Phyrok) [inline connectivity/cloudflare-warp.md:36]: Remove \

**Review by Phyrok — CHANGES_REQUESTED** [review]
Remove multiple " \ " from document.

What are the changes to the language.md?

File: .gitbook/assets/cloudflare3.png
Comment (Phyrok) [inline .gitbook/assets/cloudflare3.png:1]: image

Try to make images the same size

File: .gitbook/assets/cloudflare4 (1).png
Comment (Phyrok) [inline .gitbook/assets/cloudflare4 (1).png:1]: duplicate

File: .gitbook/assets/cloudflare5.gif
Comment (Phyrok) [inline .gitbook/assets/cloudflare5.gif:1]: if possible remove black border on top & right

File: languages.md
Comment (Phyrok) [inline languages.md:1]: What changes have been made here?

**Comment by livingflore** [comment]
no need in it

**Pull request closed without merging**